### ISSUE TITLE
specs 1.4.4 depends on junit (must not)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [etorreborre/specs#70]
```
Under 1.4.3 my program compiles fine, under 1.4.4 I get an error:

   [scalac] Compiling 38 source files to
/vol0/nga/devel/yandex/mysql-diff/target/classes
   [scalac] error: error while loading Specification, class file
'/home/nga/.ivy2/cache/org.scala-tools.testing/specs/jars/specs-1.4.4.jar(org/sp
ecs/Specification.class)'
is broken
   [scalac] (class org.junit.runner.RunWith not found.)
   [scalac]
/vol0/nga/devel/yandex/mysql-diff/src/main/scala/ru/yandex/mysqlDiff/diff/DiffMa
ker.scala:241:
error: illegal inheritance;
   [scalac]  self-type ru.yandex.mysqlDiff.diff.DiffMakerTests.type does
not conform to org.specs.Specification's selftype org.specs.Specification
   [scalac] object DiffMakerTests extends org.specs.Specification {
   [scalac]                                         ^

```

Original issue reported on code.google.com by `stepan.k...@gmail.com` on 26 Apr 2009 at 2:40

**Comment by GoogleCodeExporter** [comment]
```
I'm sorry about this. My goal is to allow a specification to be run with JUnit 
just 
by adding a JUnit trait.

I thought that Scala had an issue with inherited annotations which was 
preventing the 
@RunWith annotation on the JUnit trait to be inherited on the Specification 
class.  
So I put this annotation on the Specification class temporarily. However, this 
behavior is part of the Java specification for inherited annotations 
(http://java.sun.com/javase/6/docs/api/java/lang/annotation/Inherited.html).

So I need to find another way to run specifications with JUnit.
```

Original comment by `etorrebo...@gmail.com` on 27 Apr 2009 at 5:14

**Comment by GoogleCodeExporter** [comment]
```
This issue is fixed in 1.5.1-SNAPSHOT. 

Now there is a special class which has the proper trait *and* annotation for 
JUnit: 
SpecificationWithJUnit.
```

Original comment by `etorrebo...@gmail.com` on 21 May 2009 at 3:44
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `etorrebo...@gmail.com` on 14 Feb 2010 at 2:43
- Changed state: **Verified**
